packages/server/graphql/types/EndTeamPromptPayload.ts

```diff
@@ -0,0 +1,30 @@
+import {GraphQLID, GraphQLNonNull, GraphQLObjectType} from 'graphql'
+import {GQLContext} from '../graphql'
+import makeMutationPayload from './makeMutationPayload'
+import Team from './Team'
+import TeamPromptMeeting from './TeamPromptMeeting'
+
+export const EndTeamPromptSuccess = new GraphQLObjectType<any, GQLContext>({
+  name: 'EndTeamPromptSuccess',
+  fields: () => ({
+    meeting: {
+      type: new GraphQLNonNull(TeamPromptMeeting),
+      resolve: ({meetingId}: {meetingId: string}, _args: unknown, {dataLoader}) => {
```

Comment:
+1 We could use `resolve: resolveNewMeeting` here https://github.com/ParabolInc/parabol/blob/133ddb94ac90220ec2c8703470b274127f23a250/packages/server/graphql/resolvers.ts#L29

In the future we probably want to use just `resolve: (source) => source` here and do all the loading in `TeamPromptMeeting` but that's out of scope for now.